presto-tests/src/main/java/io/prestosql/tests/AbstractTestDistributedQueries.java

```diff
@@ -1053,6 +1054,20 @@ public void testViewAccessControl()
         assertAccessAllowed(nestedViewOwnerSession, "DROP VIEW test_nested_view_access");
         assertAccessAllowed(viewOwnerSession, "DROP VIEW test_view_access");
         assertAccessAllowed(viewOwnerSession, "DROP VIEW test_invoker_view_access");
+
+        // show create view permissions
+        @Language("SQL") String queryGrantView = "SELECT BIGINT '123' x, 'foo' y";
+        assertUpdate("CREATE VIEW meta_test_grant_access_view AS " + queryGrantView);
+
+        @Language("SQL") String queryDeniedView = "SELECT BIGINT '123' x, 'foo' y";
+        assertUpdate("CREATE VIEW meta_test_denied_access_view AS " + queryDeniedView);
+
+        // change access denied exception to view
+        assertAccessDenied("SHOW CREATE VIEW meta_test_denied_access_view", "Cannot show columns of .*meta_test_denied_access_view.*", privilege("meta_test_denied_access_view", SHOW_COLUMNS));
+        assertAccessAllowed("SHOW CREATE VIEW meta_test_grant_access_view", privilege("meta_test_denied_access_view", SHOW_COLUMNS));
+
+        assertUpdate("DROP VIEW meta_test_grant_access_view");
+        assertUpdate("DROP VIEW meta_test_denied_access_view");
```

Comment:
I don't understand why you need two views. Also can you use existing views like `test_view_access`.

Reply:
done